dd-java-agent/instrumentation/aws-java-sfn-2.0/src/main/java/datadog/trace/instrumentation/aws/v2/sfn/InputAttributeInjector.java

```diff
@@ -0,0 +1,49 @@
+package datadog.trace.instrumentation.aws.v2.sfn;
+
+import datadog.json.JsonMapper;
+import datadog.json.JsonWriter;
+import datadog.trace.bootstrap.instrumentation.api.AgentSpan;
+
+public class InputAttributeInjector {
+  private static final String DATADOG_KEY = "_datadog";
+
+  public static String buildTraceContext(AgentSpan span) {
+    String tagsJson = JsonMapper.toJson(span.getTags());
+    try (JsonWriter writer = new JsonWriter()) {
+      writer.beginObject();
+      writer.name("x-datadog-trace-id").value(span.getTraceId().toString());
+      writer.name("x-datadog-parent-id").value(String.valueOf(span.getSpanId()));
+      writer.name("x-datadog-tags").jsonValue(tagsJson);
+      writer.endObject();
+      return writer.toString();
+    } catch (Exception e) {
+      return "{}";
```

Comment:
```suggestion
      return null;
```
to match the check in `getModifiedInput` to leave the request unmodified